phpBB/language/en/composer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "en",
```

Comment:
Well we want to be able to know the name, so you don't have to look it up first when you install a forum + language via composer, so yes  a naming convention makes sense